### ISSUE TITLE
Installing on Windows with `defaults` channel enabled can be a licensing problem

### ISSUE THREAD

**Issue by nialov** [GispoCoding/eis_toolkit#433]
Anaconda has a license that makes using `defaults` channel problematic for organizations above 200 people (<https://www.anaconda.com/blog/is-conda-free#summarize>). I will make a PR removing the guidance to use the `defaults` channel. This will mean we cannot provide guidance on installing on Windows using `conda` with `defaults` channel as there is a high risk for users to breach the license unwittingly.

This is not a problem for Linux as all packages are available on `conda-forge`. However, for Windows, `tensorflow` is not maintained in `conda-forge`.

This removes the only tested installation method for Windows for `eis_toolkit`...

**Comment by nmaarnio** [comment]
Good that you noticed this license issue, I was not aware of it.

`tensorflow` is currently used only for the MLP models which are available also in `sklearn`. There were some reasons why we decided to include `tensorflow` but I cannot recall them now. @msmiyels , do you remember and what do you think about changing our MLP implementations to use `sklearn` instead of `tensorflow`? Can the BNNs you are working on be implemented with `sklearn`?

When it comes to tested installation methods on Windows, so far EIS testers in our workshops have used `pip` and installed EIS Toolkit into a `venv`. We found out some Python versions that made the installation run without issues.

I'll go approve and merge the PR, but I'd be interested to know if we could manage without the packages that are not available in `conda-forge`.

**Comment by nialov** [comment]
> When it comes to tested installation methods on Windows, so far EIS testers in our workshops have used `pip` and installed EIS Toolkit into a `venv`. We found out some Python versions that made the installation run without issues.

Yes, I remember you using this alternative method. I have been the one advocating for `conda` 😬 thinking that it would be simpler. But as long as we are using `tensorflow`, `conda` provides no good/simpler ways of installing on Windows.

If it turns out we can drop `tensorflow` from the dependencies, let me know and I will check that there are no other missing Windows packages and that I can get the environment working, Only then should `eis_toolkit` code be refactored so no one wastes time.

**Comment by nialov** [comment]
> I understood that this is only an issue if the packages come from the **default** repo, so would it be worth to try it with `pip` instead ❓ Can be accomplished by adding it to the `.yml` like
> 
> ```python
> dependencies:
>  - numpy==x.xx.xx
>  - ...
>  - pip:
>     - tensorflow==x.xx.x
>     - tensorflow_probability==x.xx.x
>     - ...
> ```

This is possible for the `environment.yaml` file in `eis_toolkit` but not in the package definition for `conda-forge`. Would be difficult to keep consistent, test, etc. ☹️
